lib/tessel/access-point.js

```diff
@@ -46,14 +47,46 @@ Tessel.prototype.enableAccessPoint = function() {
 Tessel.prototype.disableAccessPoint = function() {
   var status = 'Access Point successfully disabled.';
 
-  return new Promise((resolve) => {
+  return new Promise((resolve, reject) => {
     return this.simpleExec(commands.turnAccessPointOff())
       .then(() => {
-        return commitAndClose(this, status, resolve);
+        return commitAndClose(this, status, resolve, reject);
       });
   });
 };
 
+Tessel.prototype.getAccessPointInfo = function() {
+  var info;
+
+  return new Promise((resolve, reject) => {
+    return this.simpleExec(commands.getAccessPointConfig())
+      .then((config) => {
+        const props = ['ssid', 'key', 'encryption', 'disabled'];
+        const values = props.map((prop) => {
+          const regex = new RegExp(`${prop}='(.+)'`);
```

Comment:
This pattern, but not this specific case, can potentially be a security hole. To be clear, the point im making is absolutely irrelevant to your changes here, I just wanted to use the opportunity as illustration.